i18n/en.json

```diff
@@ -1744,6 +1744,7 @@
   "app.channel.post_update_channel_purpose_message.removed": "{username} removed the channel purpose (was: {old})",
   "app.channel.post_update_channel_purpose_message.updated_from": "{username} updated the channel purpose from: {old} to: {new}",
   "app.channel.post_update_channel_purpose_message.updated_to": "{username} updated the channel purpose to: {new}",
+  "applications": "Applications",
```

Comment:
This must be alphabetically ordered in the file, if not, the CI is going to complain.

Reply:
thanks!

Reply:
it seems that it is still complaining though :-(

Reply:
Yes, is still complaining because you changed the `defaultMessage` instead of the `id`

Reply:
not quite following. What does the HTML ID have to do with the translations?
I changed the ID

Reply:
It is not an HTML `id` is a React component property called `id` which have no relation at all with the HTML `id` attribute.

Reply:
ah, my lack of knowledge of react shines :-)
The CI seems to be passing now